### PR TITLE
book merge: Update bookmarks

### PR DESCRIPTION
**Pull request by henczati**
Corresponds to website content as of 2018-05-20.

Note: `TOC.html` lists *Index* from p. 773 while it currently starts from p. 774.

### CODE REVIEW

File: deep_learning_book/deep_learning_book.tex [hunk 1]
```diff
@@ -40,196 +40,196 @@
...
+\bookmark[page=216,view={XYZ 0 \calc{\paperheight} null},level=2]{6.5 Back-Propagation and Other Differentiation Algorithms}
+\bookmark[page=236,view={XYZ 0 \calc{\paperheight} null},level=2]{6.6 Historical Notes}
+\bookmark[page=240,view={XYZ 0 \calc{\paperheight} null},level=1]{7 Regularization for Deep Learning}
+\bookmark[page=242,view={XYZ 0 \calc{\paperheight} null},level=2]{7.1 Parameter Norm Penalties}
+\bookmark[page=249,view={XYZ 0 \calc{\paperheight} null},level=2]{7.2 Norm Penalties as Constrained Optimization}
+\bookmark[page=251,view={XYZ 0 \calc{\paperheight} null},level=2]{7.3 Regularization and Under-Constrained Problems}
+\bookmark[page=252,view={XYZ 0 \calc{\paperheight} null},level=2]{7.4 Dataset Augmentation}
+\bookmark[page=254,view={XYZ 0 \calc{\paperheight} null},level=2]{7.5 Noise Robustness}
+\bookmark[page=256,view={XYZ 0 \calc{\paperheight} null},level=2]{7.6 Semi-Supervised Learning}
+\bookmark[page=257,view={XYZ 0 \calc{\paperheight} null},level=2]{7.7 Multitask Learning}
```
Comment (henczati) [inline deep_learning_book/deep_learning_book.tex:111]: Section name fixed.

File: deep_learning_book/deep_learning_book.tex [hunk 2]
```diff
@@ -40,196 +40,196 @@
...
+\bookmark[page=257,view={XYZ 0 \calc{\paperheight} null},level=2]{7.8 Early Stopping}
+\bookmark[page=265,view={XYZ 0 \calc{\paperheight} null},level=2]{7.9 Parameter Tying and Parameter Sharing}
+\bookmark[page=267,view={XYZ 0 \calc{\paperheight} null},level=2]{7.10 Sparse Representations}
+\bookmark[page=269,view={XYZ 0 \calc{\paperheight} null},level=2]{7.11 Bagging and Other Ensemble Methods}
+\bookmark[page=271,view={XYZ 0 \calc{\paperheight} null},level=2]{7.12 Dropout}
+\bookmark[page=281,view={XYZ 0 \calc{\paperheight} null},level=2]{7.13 Adversarial Training}
+\bookmark[page=283,view={XYZ 0 \calc{\paperheight} null},level=2]{7.14 Tangent Distance, Tangent Prop and Manifold Tangent Classifier}
```
Comment (henczati) [inline deep_learning_book/deep_learning_book.tex:118]: Section name fixed.

**Comment by henczati** [comment]
As marked, the commit also contains 2 section name fixes at lines [111](https://github.com/ufal/npfl114/pull/32/files#diff-61e95fd64762628a742afee81be6e35cR111) and [118](https://github.com/ufal/npfl114/pull/32/files#diff-61e95fd64762628a742afee81be6e35cR118) (see [TOC.html](http://www.deeplearningbook.org/contents/TOC.html)).

**Comment by foxik** [comment]
Thanks!

**Pull request merged**